pubsub/kubemq/kubemq_eventstore.go

```diff
@@ -0,0 +1,199 @@
+package kubemq
```

Comment:
On this file I have the same feedback I had on `kubemq_events`

Reply:
Changed as requested for the file kubemq_events_store